### ISSUE TITLE
MatterProposal_ApproverEdit WCAG 1.4.3: Elements must have sufficient color contrast (span)

### ISSUE THREAD

**Issue by hash-logic** [hash-logic/issue#14]
**Issue**: `Elements must have sufficient color contrast` ([`color-contrast`](https://dequeuniversity.com/rules/axe/3.2/color-contrast?application=msftAI))

**Target application**: [Matter Proposal · Custom Portal](https://mymatterssit.microsoftcrmportals.com/MatterProposal_ApproverEdit/?id=4ff75df3-0161-e911-b048-00155dd3f137)

**Element path**: #section3 > .panel-heading[title="Click\ to\ collapse\ or\ expand"] > .gridheading > span

**Snippet**:

    <span style="color:grey;font-size: 11px !important;">(mgmt approval may be required in the US)</span>

**How to fix**:

    Fix any of the following:
      Element has insufficient color contrast of 3.94 (foreground color: #808080, background color: #ffffff, font size: 8.3pt, font weight: normal). Expected contrast ratio of 4.5:1

**Environment**:
Chrome version 73.0.3683.103

====

This accessibility issue was found using Accessibility Insights for Web 2.1.2 (axe-core 3.2.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.
